data-validation/multiple-coordinate-schemas/README.md

```diff
@@ -203,3 +203,14 @@ To upload `policy.json` to the broker, run the following command:
 ```bash
 curl -X POST --data @policy.json -H "Content-Type: application/json" http://localhost:8888/api/v1/data-validation/policies
 ```
+
+## Quality Metric
+Check out the script `generate-random-data.sh` that continously generates random GPS coordinates according to the `usa-coordinates` schema.
+However, some of them are invalid. Since we added a metric function to count invalid and valid messages a quality metric can be easily defined.
```

Comment:
```suggestion
However, some of them are invalid. Since we added a metric function to count invalid and valid messages, a quality metric can be easily defined.
```